retrofit-converters/protobuf/src/main/java/retrofit/converter/ProtoConverter.java

```diff
@@ -24,30 +24,31 @@
       throw new IllegalArgumentException("Expected a protobuf message but was " + c.getName());
     }
 
-    String mimeType = body.mimeType();
```

Comment:
It's inconsistent with the other converters. We never check JSON or XML for them.